Backend/starter_project_mavericks/BlogApp/BlogApp.Application/Contracts/Persistence/IUnitOfWork.cs

```diff
@@ -8,6 +8,7 @@ namespace BlogApp.Application.Contracts.Persistence
 {
     public interface IUnitOfWork : IDisposable
     {
+        IRatingRepository RatingRepository { get; }
         I_IndexRepository _IndexRepository { get; }
         Task Save();
```

Comment:
There might be some errors that could happen while saving changes to data base and if so it would be greate if we have some way knowing whether a change happened in the database or not.  You could change this part to Task<int> Save();

and in the implementation of Unit of work in the Persistence layer add the following method instead of  public async Task Save():
public async Task<int> Save()
        {
            await _context.SaveChangesAsync();
            return await _context.SaveChangesAsync();
        }